### ISSUE TITLE
Richiesta completamento visualizzazione argomenti per data  

### ISSUE THREAD

**Issue by tommasiangelo01** [scaforchio/LAMPSchool#46]
Per quanto riguarda la visualizzazione argomenti per data, ho pensato che si potrebbe aggiungere oltre  alla  colonna "materia, argomenti e attività "  anche " docente e data di consegna compiti ",in modo da essere più completa : l'inserimento colonna docente servirebbe ai genitori in modo da avere famigliarità con i docenti e le materie che insegnano, mentre la data di scadenza servirebbe per non creare equivoci che tante volte si verificano scrivendo in modo sbagliato sul diario cartaceo .Si potrebbe aggiungere queste due colonne docenti e data scadenza o consegna compito?

**Comment by vittodevit** [comment]
Docente è fattibile, vedo di aprire PR, per quanto riguarda data di scadenza compiti non è prevista nel database e richiederebbe troppo tempo aggiungerla.

**Comment by vittodevit** [comment]
![image](https://user-images.githubusercontent.com/58268988/139733371-66dd594b-03a3-4fd9-bc31-8a763fc2e48f.png)
Funzionalità in corso di implementazione.

**Comment by vittodevit** [comment]
Feature implementata, vedi PR #47 
@tommasiangelo01 Aspettiamo merge!

**Comment by tommasiangelo01** [comment]
Grazie mille!!

Il giorno lun 1 nov 2021 alle ore 21:08 Vittorio Lo Mele <
***@***.***> ha scritto:

**Comment by scaforchio** [comment]
@vittodevit Hai previsto la compresenza di più docenti nella stessa ora?

**Comment by vittodevit** [comment]
> @vittodevit Hai previsto la compresenza di più docenti nella stessa ora?

No, perchè dalla tabella delle lezoni posso solo prelevare `iddocente int(11)` che corrisponde ad un solo docente, quindi non ho idea di come fare la compresenza.
```sql
CREATE TABLE IF NOT EXISTS tbl_lezioni (
  idlezione int(11),
  idclasse int(11),
  datalezione date,
  iddocente int(11),
  idmateria int(11),
  idlezionegruppo int(11) DEFAULT NULL COMMENT 'Se vuota lezione normale',
  numeroore smallint(6),
  orainizio smallint(6),
  argomenti varchar(512),
  attivita varchar(512),
  oraultmod timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

**Comment by scaforchio** [comment]
Tutti i docenti che hanno partecipato alla lezione li trovi nella tabella tbl_firme. L'iddocente che trovi nella tabella delle lezioni è quello che ha inserito per primo la lezione.

**Comment by vittodevit** [comment]
> Tutti i docenti che hanno partecipato alla lezione li trovi nella tabella tbl_firme. L'iddocente che trovi nella tabella delle lezioni è quello che ha inserito per primo la lezione.

Allora vado a fare una query prendendomi id lezione e prelevandomi tutte le firme? Comunque se vuoi posso fare un salto in laboratorio anche per parlare di #32

**Comment by scaforchio** [comment]
Io ora sono in laboratorio.

**Comment by tommasiangelo01** [comment]
Buongiorno ,grazie per aver implementato anche i nomi dei docenti nella visualizzazione argomenti per data;ci sarebbe solo un problema:ho verificato che in caso di supplenza , il campo materia rimane bianco nonostante il docente abbia firmato la supplenza come materia . Allego foto .Un altro problema riguarda le schede di valutazioni del primo quadrimestre
,in quanto queste riportano le assenze totali ma non le assenze a singola materia 

https://drive.google.com/file/d/1q6XkDIinLh5VjwVkFHDOYWBM1hefjmDU/view?usp=sharing

**Comment by vittodevit** [comment]
Il primo problema è stato sistemato, vedi #52.
Per quanto riguarda le supplenze verifica, dopo il merge se il problema si sistema.

**Comment by tommasiangelo01** [comment]
Si, purtroppo per quanto riguarda le supplenze ,il problema si ripresenta anche dopo il merge

**Comment by vittodevit** [comment]
> Si, purtroppo per quanto riguarda le supplenze ,il problema si ripresenta anche dopo il merge

Il merge della PR di fix non è ancora stato fatto, appena #52 viene mergeata aggiornaci!

**Comment by tommasiangelo01** [comment]
Circa qualche settimana fa Pietro Tamburrano mi aveva detto che i genitori potevano giustificare direttamente dal loro portale nella sezione "Comunicazioni scuola-famiglia" ebbene sì , ho effettuato l'accesso come tutor per vedere e ho avuto anche numerose conferme da parte dei genitori i quali mi hanno detto che a loro non risulta nessuna possibilità si giustifica

**Comment by vittodevit** [comment]
> Circa qualche settimana fa Pietro Tamburrano mi aveva detto che i genitori potevano giustificare direttamente dal loro portale nella sezione "Comunicazioni scuola-famiglia" ebbene sì , ho effettuato l'accesso come tutor per vedere e ho avuto anche numerose conferme da parte dei genitori i quali mi hanno detto che a loro non risulta nessuna possibilità si giustifica

Delle giustifiche non me ne intendo, comunque, per semplificare la tracciabilità, ogni problema mettilo su un'issue diverso.

**Comment by tommasiangelo01** [comment]
Lo faccio subito

Il giorno mer 3 nov 2021 alle ore 15:15 Vittorio Lo Mele <
***@***.***> ha scritto:

**Comment by scaforchio** [comment]
OK. Verificato.
